apps/condo/domains/property/schema/Property.js

```diff
@@ -179,6 +180,21 @@ const Property = new GQLListSchema('Property', {
                 return data.inwork.count
             },
         },
+
+        yearOfConstruction: {
+            schemaDoc: 'Year of the property was built',
+            type: CalendarDay,
+            format: 'YYYY',
+            yearRangeTo: dayjs().year(),
+        },
+
+        area: {
```

Comment:
Usually it's more than one area. Living area, Non-living area. What is this area?

Reply:
@pahaz That field contain total area of property